src/GuessThePokemon.js

```diff
@@ -1,6 +1,7 @@
 const { EmbedBuilder, AttachmentBuilder } = require('discord.js');
 const fetch = require('node-fetch');

```

Comment:
I know this line was already here, but you can remove it, the code still works without it and we can remove this as a dependency later